packages/easy-coding-standard/packages/snippet-formatter/tests/Markdown/Fixture/inline_rector.md

```diff
@@ -0,0 +1,40 @@
+Move value object to ValueObject namespace/directory
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Rector\Autodiscovery\Rector\FileSystem\MoveValueObjectsToValueObjectDirectoryRector;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+
+    $services->set(MoveValueObjectsToValueObjectDirectoryRector::class)
+        ->call('configure', [[MoveValueObjectsToValueObjectDirectoryRector::TYPES => ['ValueObjectInterfaceClassName'], MoveValueObjectsToValueObjectDirectoryRector::SUFFIXES => ['Search'], MoveValueObjectsToValueObjectDirectoryRector::ENABLE_VALUE_OBJECT_GUESSING => true]]);
+};
+```
+-----
+
+Move value object to ValueObject namespace/directory
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Rector\Autodiscovery\Rector\FileSystem\MoveValueObjectsToValueObjectDirectoryRector;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+
+    $services->set(MoveValueObjectsToValueObjectDirectoryRector::class)
+        ->call('configure', [[
+            MoveValueObjectsToValueObjectDirectoryRector::TYPES => ['ValueObjectInterfaceClassName'],
```

Comment:
@samsonasik Hi, I found out an issue with markdown snippet formatter.

This file is skipped for some reason. It should be indented on new line:

```diff
-->call('configure', [[Move...
+->call('configure', [[
+Move...
```

Could you check it? Feel free to copy this file to new pull-request

Reply:
ok, I will check.

Reply:
it seems the issue is on regex part, see https://regex101.com/r/4YUIu1/6